Pages/Index.cshtml

```diff
@@ -11,21 +11,36 @@
 
 <h1 class="status-message">@Model.StatusMessage</h1>
 
-<form method="post">
-    <div class="form-field">
-        <label asp-for="Id"></label>
-        <input asp-for="Id"/>
-        <span asp-validation-for="Id"></span>
-    </div>
-
-    <div class="form-field">
-        <label asp-for="Control"></label>
-        <input asp-for="Control"/>
-        <span asp-validation-for="Control"></span>
-    </div>
-
-    <button type="submit" class="btn">Save</button>
-</form>
+<fieldset>
+    <legend>Form 1</legend>
+    <form method="post">
+        <div class="form-field">
+            <label asp-for="Id"></label>
+            <input asp-for="Id"/>
+            <span asp-validation-for="Id"></span>
+        </div>
+
+        <div class="form-field">
+            <label asp-for="Control"></label>
+            <input asp-for="Control"/>
+            <span asp-validation-for="Control"></span>
+        </div>
+
+        <div class="form-field">
+            <label>
+                <input asp-for="Color" /> Red
+            </label>
+
+            <label>
+                <input asp-for="Color" /> Blue
+            </label>
+            <span asp-validation-for="Color"></span>
+        </div>
```

Comment:
I added this example to show that your fix works. I couldn't get it to work for radio buttons because I couldn't get it to invalidate them. It's been so long. 😆 I'll merge this as-is. If you have a better minimal repro using radio or checkboxes, I'd appreciate a follow-up. 😄

Reply:
Thanks for reviewing and merging this in.  I'll create another PR to add a checkbox list and radio button lists to the demo web app you have in here.

Reply:
Thanks! It looks like we might have a bug with validating radio button lists in the first place: https://github.com/haacked/aspnet-client-validation/issues/36

Reply:
Please find PR here: https://github.com/haacked/aspnet-client-validation/pull/43